### PR TITLE
1. add link to info-func 2. add doc for function reference

### PR DESCRIPTION
**Pull request by XuHuaiyu**
PTAL
@shenli

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: sql/functions-and-operators-reference.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: TiDB 用户文档
+category: user guide
+---
+
+TiDB 中函数和操作符使用方法与 MySQL 基本一致, 详情参见: [Functions and Operators](https://dev.mysql.com/doc/refman/5.7/en/functions.html)
+
+# 函数和操作符概述
```
Comment (andelf) [inline sql/functions-and-operators-reference.md]: move heading to line 6

**Comment by shenli** [comment]
@XuHuaiyu Any update?

**Comment by XuHuaiyu** [comment]
PTAL @andelf

**Pull request merged**